docs/community/contribute/discussion.md

```diff
@@ -42,6 +42,7 @@ If you find a problem, it'll be nice to fix it: [How to contribute to a GitHub s
 ### GitLab Advisory Database
 Visit [here](https://advisories.gitlab.com/) and search CVE-ID.
 
+<!-- Please note that the link to [Create an issue to GitLab Advisory Database](https://gitlab.com/gitlab-org/security-products/gemnasium-db/-/issues/new) requires login. -->
 If you find a problem, it'll be nice to fix it: [Create an issue to GitLab Advisory Database](https://gitlab.com/gitlab-org/security-products/gemnasium-db/-/issues/new)
```

Comment:
Instead of commenting it out (users can't see this) we can do something like the following:

```suggestion
If you find a problem, it'll be nice to fix it: [Create an issue to GitLab Advisory Database](https://gitlab.com/gitlab-org/security-products/gemnasium-db/-/issues/new). Please note this requires a GitLab account.
```